apis/v1/objectreference.go

```diff
@@ -8,5 +8,6 @@ package v1
 type LocalObjectReference struct {
```

Comment:
Technically not a *Local*ObjectReference anymore, but changing this will be a breaking change so let's leave that. But please update the comment above, it's wrong now.

Reply:
updated description field